doc/source/index.rst

```diff
@@ -5,7 +5,7 @@
 .. note:: Adapting to a new version?
 
    If you are adapting to a new version of the actions, please refer to the
```

Comment:
```suggestion
   If you are adapting to a new version of the actions, see the
```
Omit "please" in technical doc. Also, use "see the" rather than "refer to the".